build/Dockerfile

```diff
@@ -81,6 +81,37 @@ RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode
 # COPY build/*.crt  /usr/local/share/ca-certificates/
 # RUN update-ca-certificates
 
+############################################# Base image for Debian with NGINX Plus and App Protect Dos #############################################
+FROM debian-plus as debian-plus-dos
+ARG IC_VERSION
+ARG NGINX_PLUS_VERSION
+
+RUN --mount=type=secret,id=nginx-repo.crt,dst=/etc/ssl/nginx/nginx-repo.crt,mode=0644 \
+	--mount=type=secret,id=nginx-repo.key,dst=/etc/ssl/nginx/nginx-repo.key,mode=0644 \
+	set -x \
+	&& apt-get update \
+	&& apt-get -y install lsb-release ca-certificates \
```

Comment:
There's no need to install `lsb-release`, you can get the name from `/etc/os-release` like in the other images and save 25MB from the final image.

Reply:
fixed

Reply:
> lsb-release is still got installed. is it necessary? same for debian-plus-nap-dos

@soneillf5

Reply:
fixed